my_sql.md

```diff
@@ -0,0 +1,37 @@
+# MySQL
+
+
+## Prerequisite knowledge:   
+DATABASES
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+* I know what kind of database MySQL is and know it's pros and cons
+* I can create a database
+* I can view database structure with basic commands
+* I know MySQL data types
+* I can create, update and delete records in the database
```

Comment:
what about `read`?

Reply:
I would add one more point about users in MySQL and granting access. This was the biggest problem for me when I was learning.